test/streams.js

```diff
@@ -41,10 +41,6 @@ function pumpifyError() {
   var read = fs.createReadStream(exists);
   var pipeline = pumpify(through(), through(withErr), through());
 
-  pipeline.on('error', function (err) {
```

Comment:
This looks like it was a workaround for the end-of-stream problem for normal streams.